specs/phase1/validator.md

```diff
@@ -536,6 +543,53 @@ class SignedLightAggregateAndProof(Container):
     signature: BLSSignature
 ```
 
+## Data Availability Sampling
```

Comment:
I would argue that this does  *not* go in the validator guide as it is functionality that non-validator nodes will use.
Instead, it should be in a DAS guide

Reply:
Fine with me, I'll move it into a new doc.